### PR TITLE
Correction on custom view using spanish

### PR DESCRIPTION
**Pull request by adrien2612**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [x] 18.10.x
- [X] 19.04.x
- [X] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [X] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by lpinsivy** [comment]
@adrien2612 you can compile and test your translation on a Centreon using

msgfmt messages.po  -o messages.mo
cp messages.mo /usr/share/centreon/www/locale/es_ES.UTF-8/LC_MESSAGES/messages.po
chown apache:apache /usr/share/centreon/www/locale/es_ES.UTF-8/LC_MESSAGES/messages.po
systemctl restart httpd24-httpd

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1286,9 +1286,8 @@ msgstr "aplicar"
 
 #: centreon-web/www/install/smarty_translate.php:159 centreon-web/www/include/home/customViews/views.php:115
 msgid "Deleting this view might impact other users. Are you sure you want to do it?"
-msgstr ""
-"Eliminar esta vista podría afectar a otros usuarios. \n"
-"¿Estás seguro de que quieres hacer eso?"
+msgstr "Eliminar esta vista podría afectar a otros usuarios."
```
Comment (sc979) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Eliminar esta vista podría afectar a otros usuarios.\\n"
```

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1286,9 +1286,8 @@ msgstr "aplicar"
...
+msgstr "¿Estás seguro de que quieres hacer eso?"
```
Comment (sc979) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
"¿Estás seguro de que quieres hacer eso?"
```

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -5392,9 +5391,8 @@ msgstr ""
 
 #: centreon-web/www/include/home/customViews/views.php:114
 msgid "Deleting this widget might impact users with whom you are sharing this view. Are you sure you want to do it?"
-msgstr ""
-"La eliminación de este widget podría afectar a los usuarios con los que comparte esta vista. \n"
-"¿Estás seguro de que quieres hacer eso?"
+msgstr "La eliminación de este widget podría afectar a los usuarios con los que comparte esta vista."
```
Comment (sc979) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
msgstr "La eliminación de este widget podría afectar a los usuarios con los que comparte esta vista.\\n"
```

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -5392,9 +5391,8 @@ msgstr ""
...
+msgstr "¿Estás seguro de que quieres hacer eso?"
```
Comment (sc979) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po]:
```suggestion
"¿Estás seguro de que quieres hacer eso?"
```

**Pull request merged**